features/GEM005_Geometry-of-surrounding-buildings.feature

```diff
@@ -0,0 +1,12 @@
+@implementer-agreement
+@ALS
+Feature: Building shape representation
+  Scenario: Agreement on empty IfcBuilding using correct representation
+
+      Given A file with Schema Identifier "IFC2X3" or "IFC4" or "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
+      And An IfcBuilding
+      And ContainsElements = empty
+      And IsDecomposedBy = empty
```

Comment:
```suggestion
      And Its attribute ContainsElements is empty
      And Its attribute IsDecomposedBy is empty
```
Using symbols defeats the purpose of having human readable sentences.

Reply:
I agree. That would be more readable. 

This discussion spans across multiple pull requests. Since the remaining implementation of this pull request are good, I'd suggest to move the discussion there and merge this PR. 
https://github.com/buildingSMART/ifc-gherkin-rules/issues/90